CHANGELOG_PENDING.md

```diff
@@ -18,6 +18,8 @@
 - [cli/import] - Code generation in `pulumi import` can now be disabled with the `--generate-code=false` flag.
   [#9141](https://github.com/pulumi/pulumi/pull/9141)
 
+- [language/dotnet] - updated Pulumi dotnet packages to use grpc-dotnet instead of grpc [#9149](https://github.com/pulumi/pulumi/pull/9149)
```

Comment:
```suggestion
- [language/dotnet] - Updated Pulumi dotnet packages to use grpc-dotnet instead of grpc.
   [#9149](https://github.com/pulumi/pulumi/pull/9149)
```